news/2019-09-27-mappers-guild-september-newspost.md

```diff
@@ -0,0 +1,131 @@
+- [Imperial Circus Dead Decadence - Shinbatsu o Tadori Kyoukotsu ni Itaru](https://osu.ppy.sh/beatmapsets/482382) by [Kite](https://osu.ppy.sh/users/134572)
+
+The **ICDD: Kuruoshiku Saita Seisan na Mukuro wa Kanade, Itooshiku Saita Shoujo wa Seisen no Kotoba wo Utau** quest got completed by [VINXIS](https://osu.ppy.sh/users/4323406), [deetz](https://osu.ppy.sh/users/3513559), [RLC](https://osu.ppy.sh/users/1047883), [Nyquill](https://osu.ppy.sh/users/682935), [Cheesecake](https://osu.ppy.sh/users/3700647) and [Sing](https://osu.ppy.sh/users/3795679).
+
+- [Imperial Circus Dead Decadence - Uta](https://osu.ppy.sh/beatmapsets/962562) by [deetz](https://osu.ppy.sh/users/3513559)
+- [Imperial Circus Dead Decadence - Ankoku No Jyoukaku Ni Shinkou Saru Igyou No Kyouki](https://osu.ppy.sh/beatmapsets/967766) by [Cheesecake](https://osu.ppy.sh/users/3700647)
+- [Imperial Circus Dead Decadence - Tinu rareta Syuren Ni Fukeru Homura](https://osu.ppy.sh/beatmapsets/964134) by [VINXIS](https://osu.ppy.sh/users/4323406)
+- [Imperial Circus Dead Decadence - Syakunetsu No Rengoku Ni Bara Wa...](https://osu.ppy.sh/beatmapsets/964680) by [Sing](https://osu.ppy.sh/users/3795679)
+
+![sakuraburst header image](https://assets.ppy.sh/artists/8/header.jpg)
+
+The **Neuro Genre Showcase** quest got completed by **[Nifty](https://osu.ppy.sh/users/4956097)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)**, **[Faputa](https://osu.ppy.sh/users/845733)**, **[\_DUSK\_](https://osu.ppy.sh/users/6092181)**, **[Yales](https://osu.ppy.sh/users/2377881)**, **[Nepuri](https://osu.ppy.sh/users/6637817)**, **[Tyistiana](https://osu.ppy.sh/users/1421452)** and **[aabc271](https://osu.ppy.sh/users/155707)**
+
+- [sakuraburst - cherry blossoms explode across the dying horizon](https://osu.ppy.sh/beatmapsets/878621) by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
+- [Dictate - Cauldron](https://osu.ppy.sh/beatmapsets/942334) by [Nepuri](https://osu.ppy.sh/users/6637817)
+- [sakuraburst - dragonlands](https://osu.ppy.sh/beatmapsets/935765) by [Faputa](https://osu.ppy.sh/users/845733)
+- [Culprate & Joe Ford - Gaucho](https://osu.ppy.sh/beatmapsets/936234) by [Nifty](https://osu.ppy.sh/users/4956097)
+- [sakuraburst - SELF DESTRUCT](https://osu.ppy.sh/beatmapsets/963193) by [\_DUSK\_](https://osu.ppy.sh/users/6092181)
+
+![tieff header image](https://assets.ppy.sh/artists/34/header.jpg)
+
+The **tieff Mini-pack** quest got completed by **[Mirash](https://osu.ppy.sh/users/2841009)**, **[schoolboy](https://osu.ppy.sh/users/8722791)**, **[Hinsvar](https://osu.ppy.sh/users/1249323)**, **[Realazy](https://osu.ppy.sh/users/918297)**, **[PandaHero](https://osu.ppy.sh/users/1233255)**, **[Trynna](https://osu.ppy.sh/users/2652951)**, **[eiri-](https://osu.ppy.sh/users/3388410)** and **[Delis](https://osu.ppy.sh/users/1603923)**.
```

Comment:
Delis changed their name a few days ago into ShirohaMyMommy btw : http://osu.ppy.sh/users/1603923

(i thought it would be better to change it to keep stuffs consistent since you had "type 1 if cute" instead of "bor" in Line 54 already)